oa4mp/serve.go

```diff
@@ -309,22 +309,26 @@ func ConfigureOA4MP() (launcher daemon.Launcher, err error) {
 	// Ensure the OA4MP storage directory exists and has correct permissions and ownership
 	OA4MPStoragePath := "/opt/scitokens-server/var/storage"
 
-	if err := os.Chmod(OA4MPStoragePath, 0700); err != nil {
+	if err = os.Chmod(OA4MPStoragePath, 0700); err != nil {
 		if os.IsNotExist(err) {
 			log.Debugln("OA4MP storage directory does not exist. Creating", OA4MPStoragePath)
 			if err = os.MkdirAll(OA4MPStoragePath, 0755); err != nil {
-				log.Fatal("Failed to create directory", OA4MPStoragePath, ":", err)
+				err = errors.Wrap(err, "Failed to create OA4MP storage directory")
```

Comment:
I think this is the last thing I see -- the preferred go syntax is to keep wrapped error messages lowercase, so we prefer `errors.Wrap(err, "failed to create OA4MP ...")` instead of `errors.Wrap(err, "Failed to create OA4MP ...")`. Once that's changed in these error messages, I think this is ready 😀

Reply:
ohhh my bad. on it rn

Reply:
ok. I pushed a commit to punch these capital letters @jhiemstrawisc